shimmer/develop/what-is-shimmer/layer-1-landscape.md

```diff
@@ -0,0 +1,141 @@
+[NFTs](https://wiki.iota.org/shimmer/introduction/explanations/ledger/nft/)
+and [native tokens and foundries](https://wiki.iota.org/shimmer/introduction/explanations/ledger/foundry/)
+on layer 1, as long as you can cover the
+minimum [storage deposit](#storage-deposits).
+
+## Storage Deposits
+
+### IOTA
+
+There are no data storage deposits in the IOTA network, as it relies on [dust protection](https://wiki.iota.org/introduction/reference/details/#dust-protection), which means you can only
+send microtransactions of less than 1Mi if the target address already has at least 1Mi.
+
+### Shimmer
+
+The Stardust protocol improves IOTA dust protection by
+introducing [storage deposits](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/storage_deposit/).
+You must back up any data you store on the Tangle
+with [SMR tokens](tokens-and-wallets.md#storage-deposits). **The
+deposit is not a fee**, and it can be refunded as soon as the output is cleaned up.
+
+## Output Unlock Conditions
+
+### IOTA
+
+At the time, IOTA's Chrysalis protocol only allows the
+traditional [address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#address)
+unlock condition. That is to say, outputs in Chrysalis can only be unlocked by whoever controls the keys associated with the target address.
+
+### Shimmer
+
+The Stardust protocol introduces
+new [output unlock conditions](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/)
+that allow you to specify not only
+the [unlock address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#address),
+but also:
+
+- [Storage deposit
+  returns](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#storage-deposit-return):
+  This feature allows you to specify a return amount that has to be refunded to the sender's account, which enables
+  micro-transactions of any size.
+- [Expirations](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#expiration):
+  If you define an expiration date-time for your output, it will be returned if it hasn't been claimed by said
+  date-time.
+- [Time-locks](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#timelock):
+  Conversely to expirations, you can also specify a date-time until which the recipient won't be able to claim the
+  output.
+
+## IOTA Smart Contracts (ISC)
+
+You can use [IOTA Smart Contracts (ISC)](https://wiki.iota.org/shimmer/smart-contracts/overview/) to deploy your own
+[scalable](https://wiki.iota.org/shimmer/smart-contracts/overview/#scaling-and-fees)
+and [flexible](#https://wiki.iota.org/shimmer/smart-contracts/overview/#flexibility) smart contracts to the
+IOTA ecosystem.
+
+### Ethereum Virtual Machine (EVM) Support
```

Comment:
I guess we don't want to mention wasmVM yet?

Reply:
I think we can come back to this when we add the smart contracts content. WDYT?

Reply:
Fine to me